### PR TITLE
resolving conflict

### PR DESCRIPTION
**Pull request by cwire**
upper case

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,6 @@ https://www.markdownguide.org/cheat-sheet/
 
 
 ## Topics
-* git ignore
+* Git ignore
```
Comment (badcab) [inline README.md]: shouldn't the I be caps also?

**Review by badcab — CHANGES_REQUESTED** [review]
fix this stuff

File: README.md (hunk 1 above)
Comment (cwire) [reply README.md]: fixed

Comment (badcab) [reply README.md]: https://media.giphy.com/media/RddAJiGxTPQFa/giphy.gif

Comment (badcab) [reply README.md]: giphy

**Review by badcab — CHANGES_REQUESTED** [review]
no

**Pull request merged**